tool/world_options.re

```diff
@@ -0,0 +1,300 @@
+            onclick: fn(input: string) {},
+            onchange: fn(input: string) {
+                match input.parse_float() {
+                    Result::Ok(brightness) => {
+                        Tas::set_sky_light_brightness(brightness);
+                        SETTINGS.sky_light_brightness = brightness;
+                        SETTINGS.store();
+                    },
+                    Result::Err(e) => {},
+                }
+            },
+        }),
+        UiElement::FloatInput(FloatInput {
+            label: Text { text: "Sky Light Intensity" },
+            input: f"{SETTINGS.sky_light_intensity}",
+            onclick: fn(input: string) {},
+            onchange: fn(input: string) {
+                match input.parse_float() {
+                    Result::Ok(intensity) => {
+                        Tas::set_sky_light_intensity(intensity);
+                        SETTINGS.sky_light_intensity = intensity;
+                        SETTINGS.store();
+                    },
+                    Result::Err(e) => {},
+                }
+            },
+        }),
+        UiElement::FloatInput(FloatInput {
+            label: Text { text: "Stars Brightness" },
+            input: f"{SETTINGS.stars_brightness}",
+            onclick: fn(input: string) {},
+            onchange: fn(input: string) {
+                match input.parse_float() {
+                    Result::Ok(brightness) => {
+                        Tas::set_stars_brightness(brightness);
+                        SETTINGS.stars_brightness = brightness;
+                        SETTINGS.store();
+                    },
+                    Result::Err(e) => {},
+                }
+            },
+        }),
+        UiElement::FloatInput(FloatInput {
+            label: Text { text: "Sun Color Red" },
+            input: f"{SETTINGS.sun_color_red}",
+            onclick: fn(input: string) {},
+            onchange: fn(input: string) {
+                match input.parse_float() {
+                    Result::Ok(color) => {
+                        Tas::set_sun_color(color);
+                        SETTINGS.sun_color_red = color;
+                        SETTINGS.store();
+                    },
+                    Result::Err(e) => {},
+                }
+            },
+        }),
+        UiElement::FloatInput(FloatInput {
+            label: Text { text: "Cloud Color Red" },
+            input: f"{SETTINGS.cloud_color_red}",
+            onclick: fn(input: string) {},
+            onchange: fn(input: string) {
+                match input.parse_float() {
+                    Result::Ok(color) => {
+                        Tas::set_cloud_color(color, SETTINGS.cloud_color_green, SETTINGS.cloud_color_blue);
+                        SETTINGS.cloud_color_red = color;
+                        SETTINGS.store();
+                    },
+                    Result::Err(e) => {},
+                }
+            },
+        }),
```

Comment:
Does it make sense to take a hex-RGB-string and convert it to the respective color floats?